src/swig/stir.i

```diff
@@ -984,6 +984,9 @@ T * operator-> () const;
   }
 #endif
 
+%include "stir/deprecated.h"
```

Comment:
```suggestion
```
don't let SWIG parse it.

Reply:
If I don't parse it first I get a build error ` Error: Syntax error in input(3)`

from: https://github.com/UCL/STIR/blob/f9cbaf54995caa4210c9dd080a06a8bc25abda51/src/include/stir/NumericVectorWithOffset.h#L133

Reply:
ok. I'd somewhat prefer to modify
https://github.com/UCL/STIR/blob/29f0027429c0ba7f42a8e877eeff3f957768f5b8/src/swig/CMakeLists.txt#L36
to set `DSTIR_DEPRECATED=""` but i guess if it works it's fine.

Reply:
I've updated it to unset the macro, thanks for pointing out the line. I'm a rabbit in headlights when I try to read cmake files!